packages/qwik/src/core/state/common.ts

```diff
@@ -489,11 +489,20 @@ export class LocalSubscriptionManager {
           }
         } else {
           const signal = sub[SubscriptionProp.SIGNAL];
-          scheduler.$scheduleNodeDiff$(
-            host as fixMeAny,
-            sub[SubscriptionProp.ELEMENT] as fixMeAny,
-            untrack(() => signal.value)
-          );
+          if (type == SubscriptionType.PROP_IMMUTABLE || type == SubscriptionType.PROP_MUTABLE) {
+            scheduler.$scheduleNodeProp$(
```

Comment:
I don't think a call to `scheduler` is necessary. Check my thinking if it makes sense.

The scheduler is needed because we want to ensure that the side-effects of calling components functions (`useTask` etc.) are called in depth-first order. (Because they have `use*` hooks)

But updating a prop/attr (or text) does not need to go through the scheduler because it does not have any side effects (ie no `useTask` or hooks will be invoked). (Not considering DOM update as a side-effect we care about. More specifically, the order of DOM updates does not matter.)

Because no hooks are involved, the order of updating the DOM does not matter. (as long as we do all of the updates in a single micro-task, which we do because of journal) 

So the scheduler here is just overhead, and I think it is safe to skip it, as that will gain us speed.

Reply:
While this works I need to import few things that are not allowed to import inside the ssr-container.
```ts
import { trackSignal } from '../core/use/use-core';
import { SubscriptionType } from '../core/state/common';
```